src/App.ts

```diff
@@ -1,4 +1,12 @@
 import { Server } from './Server';
+import { loadConfig } from './utils/Config';
+
+/**
+ * Load environment config.
+ */
+const config = loadConfig();
+if (config.error)
+    console.error(config.error);
```

Comment:
I am not sure what make `config.error` but I'd rather crash the app than progress with invalid configuration in the fail-fast spirit.